spec/models/members_spec.rb

```diff
@@ -10,7 +10,7 @@
 
     it { should validate_presence_of(:user) }
     it { should validate_presence_of(:source) }
-    it { should ensure_inclusion_of(:access_level).in_array(Gitlab::Access.values) }
+    it { should validate_inclusion_of(:access_level).in_array(Gitlab::Access.values) }
```

Comment:
Gonna leave this one unless you really want it fixed. Splitting it would just make it look worse I think.